internal/provider/resource_port_cloud_router_connection.go

```diff
@@ -131,10 +134,22 @@ func resourceCustomerOwnedPortConnRead(ctx context.Context, d *schema.ResourceDa
 	var diags diag.Diagnostics
 	if cid, ok := d.GetOk("circuit_id"); ok {
 		cloudConnCID := d.Get("id")
-		_, err := c.ReadAwsConnection(cid.(string), cloudConnCID.(string))
+		resp, err := c.ReadCloudRouterConnection(cid.(string), cloudConnCID.(string))
 		if err != nil {
 			diags = diag.FromErr(err)
 		}
+
+		_ = d.Set("account_uuid", resp.AccountUUID)
+		_ = d.Set("circuit_id", resp.CloudRouterCircuitID)
+		_ = d.Set("maybe_nat", resp.NatCapable)
+		_ = d.Set("maybe_dnat", resp.DNatCapable)
+		_ = d.Set("port_circuit_id", resp.PortCircuitID)
+		_ = d.Set("description", resp.Description)
+		_ = d.Set("vlan", resp.Vlan)
+		_ = d.Set("speed", resp.Speed)
+
+		_unsetFields := []string{"untagged", "is_public", "published_quote_line_uuid"}
+		showWarningForUnsetFields(_unsetFields, &diags)
```

Comment:
Now, the warning shows all the time, even when running the Terraform apply. Do you think there would be a way to only show it when the `terraform import` is running?